main.py

```diff
@@ -2,6 +2,30 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
```

Comment:
It's a good practice to specify the type of `QuerySet`, for example in this case, `QuerySet[Actor]`, to improve code clarity.